README.adoc

```diff
@@ -4,13 +4,60 @@ This project aims at making it as seamless as possible for Spring users to run t
 Google Cloud Platform. The project website is http://cloud.spring.io/spring-cloud-gcp/ (under
 development).
 
-Currently, this repository contains Spring Integration inbound and outbound channel adapters which
-use Google Cloud Pub/Sub, as well as a Google Cloud Pub/Sub Spring abstraction, i.e.,
-`PubSubTemplate`.
+Currently, this repository provides support for:
```

Comment:
Suggestion:
in Line 3, change "This project aims at making it as seamless as possible for Spring users..."
to
"This project makes it easier for Spring users..."